cg/cli/demultiplex/copy_novaseqx_data.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import os
+import shutil
+from typing import List, Optional
+
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def is_copied(analysis_directory: Path) -> bool:
+    return Path(analysis_directory, DemultiplexingDirsAndFiles.COPY_COMPLETE).exists()
+
+
+def is_analyzed(analysis_directory: Path) -> bool:
```

Comment:
Same suggestion here to be more explicit:

```suggestion
def is_flow_cell_demultiplexed(analysis_directory: Path) -> bool:
```